src/networkmessage.cpp

```diff
@@ -95,6 +95,12 @@ void NetworkMessage::addItem(uint16_t id, uint8_t count)
 		addByte(0x00); // quiver ammo count
 	} else if (it.classification > 0) {
 		addByte(0x00); // item tier (0-10)
+	} else if (it.showClientCharges) {
+		add<uint32_t>(it.charges);
+		addByte(0x00); // ??
```

Comment:
boolean "isBrandNew"

you can hide charges/duration for brand-new items in client